jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -218,7 +219,60 @@ public void testSteps_forBuildToDockerRegistry_multipleTags()
   }
 
   @Test
-  public void tesBuildToDockerRegistry_dockerHubBaseImage()
+  public void testSteps_forBuildToDockerRegistry_skipExistingDigest()
+      throws IOException, InterruptedException, ExecutionException, RegistryException,
+          CacheDirectoryCreationException {
+    System.setProperty(JibSystemProperties.SKIP_EXISTING_IMAGES, "true");
+
+    JibContainer image1 =
+        buildRegistryImage(
+            ImageReference.of("gcr.io", "distroless/java", DISTROLESS_DIGEST),
+            ImageReference.of("localhost:5000", "testimagerepo", "testtag"),
+            Collections.singletonList("testtag2"));
+
+    // Test that the initial image with the original tag has been pushed.
+    String imageReference = "localhost:5000/testimagerepo:testtag";
+    localRegistry.pull(imageReference);
+    assertDockerInspect(imageReference);
+    Assert.assertEquals(
+        "Hello, world. An argument.\n", new Command("docker", "run", "--rm", imageReference).run());
+
+    // Test that any additional tags have also been pushed with the original image.
+    String imageReference2 = "localhost:5000/testimagerepo:testtag2";
+    localRegistry.pull(imageReference2);
+    assertDockerInspect(imageReference2);
+    Assert.assertEquals(
+        "Hello, world. An argument.\n",
+        new Command("docker", "run", "--rm", imageReference2).run());
```

Comment:
I think we can skip this testing, as the target of this test method is to test `SKIP_EXISTING_IMAGES`. Our integration are increasing running more Docker images and getting slower.

Reply:
Do you mean that all the test is unnecessary, or just the part where `testtag` and `testtag2` are pushed?

Reply:
Sorry, I mean this "Hello, world" assertion on this line and and `assertDockerInpsect()` on line 243. Also remove them on line 236 and 237.